h/static/scripts/tests/controllers/lozenge-template.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// Template copied from navbar.html.jinja2
```

Comment:
For now I've just copied the server template from `navbar.html.jinja2` rather than try to actually use the same markup, although that wouldn't be super hard to do in future.